lib/jobs.js

```diff
@@ -24,6 +25,15 @@ const executor = new ExecutorRouter({
     executor: executorPlugins,
     ecosystem
 });
+const retryOptions = {
+    plugins: ['Retry'],
+    pluginOptions: {
+        Retry: {
+            retryLimit: 3,
```

Comment:
Non-blocking: define the magic numbers as constants

Reply:
👍 fixing it